db/migrate/20210303025206_add_institution_to_course.rb

```diff
@@ -0,0 +1,13 @@
+class AddInstitutionToCourse < ActiveRecord::Migration
+  def up
+    change_table :courses do |t|
+      t.references :institute
+    end
+  end
+
+  def down
+    change_table :courses do |t|
+      t.remove_references :institute
+    end
+  end
+end
```

Comment:
no need to change but I think you can simplify this for future reference (no pun intended),

```suggestion
class AddInstitutionToCourse < ActiveRecord::Migration
  def change
    add_column :courses, :institute, :references
  end
```

This can also be generated for you with 

```bash
bundle exec rails g migration add_institution_to_course institute:references
```

the migration generator is smart enough to know that the `to_course` bit in the name (or `ToCourse` if you go capital camel case) means it is adding it to the `courses` table

Reply:
Tried that one, `add_column` with `:references` doesn't work in rails 3.2.